src/components/Dropdown/components/ValueContainer/ValueContainer.jsx

```diff
@@ -9,10 +10,11 @@ import Chips from "../../../Chips/Chips";
 import classes from "./ValueContainer.module.scss";
 
 export default function Container({ children, selectProps, ...otherProps }) {
-  const { placeholder, selectProps: customProps = {} } = selectProps;
+  const { placeholder, inputValue, selectProps: customProps = {} } = selectProps;
   const { selectedOptions, onSelectedDelete, setIsDialogShown, isDialogShown, isMultiline } = customProps;
   const clickHandler = children[1];
   const [ref, setRef] = useState();
+  const showPlaceholder = useMemo(() => selectedOptions.length === 0 && !inputValue, [inputValue, selectedOptions]);
   const chipClassName = isMultiline ? classes["multiselect-chip-multi-line"] : classes["multiselect-chip-single-line"];
```

Comment:
according to our new standards we are not using use memo fro simple conditions (the cache is also has performance cost)